### ISSUE TITLE
center loss implementation

### ISSUE THREAD

**Issue by amirhfarzaneh** [KevinMusgrave/pytorch-metric-learning#108]
Is center loss considered to be a metric learning approach? If you think it can be part of this repo, I am interested in contributing.

**Comment by KevinMusgrave** [comment]
Yes that would be great! I assume you're talking about [this paper](https://ydwen.github.io/papers/WenECCV16.pdf). In terms of implementation, take a look at [NormalizedSoftmaxLoss](https://github.com/KevinMusgrave/pytorch-metric-learning/blob/master/src/pytorch_metric_learning/losses/normalized_softmax_loss.py) for an example of how a classification loss is implemented.

**Comment by amirhfarzaneh** [comment]
Yes it's the same paper. I will take a look at that example.
